src/basic/inotify-util.h

```diff
@@ -8,7 +8,7 @@
 
 #include "log.h"
 
-#define INOTIFY_EVENT_MAX (offsetof(struct inotify_event, name) + NAME_MAX + 1)
+#define INOTIFY_EVENT_MAX (sizeof(struct inotify_event) + NAME_MAX + 1)
```

Comment:
this makes no sense? the name[] array is the last field in the struct and is defined with unspecified size, hence sizeof() it will be zero. This this actually resolves to the very same thing effectively. Except that the offsetof() way to express this makes clear what is going on here: that there's a variable size field "name" at the end.